ads/README.md

```diff
@@ -67,6 +67,8 @@ More information can be provided in a similar fashion if needed (Please file an
 - `window.context.renderStart(opt_data)` is a method to inform AMP runtime when the ad starts rendering. The ad will then become visible to user. The optional param `opt_data` is an object of form `{width, height}` to request an [ad resize](#ad-resizing) if the size of the returned ad doesn't match the ad slot. To enable this method, add a line `renderStartImplemented=true` to the corresponding ad config in [_config.js](./_config.js).
 - `window.context.noContentAvailable()` is a method to inform AMP runtime that the ad slot cannot be filled. The ad slot will then display the fallback content if provided, otherwise try to collapse.
 - `window.context.reportRenderedEntityIdentifier()` MUST be called by ads, when they know information about which creative was rendered into a particular ad frame and should contain information to allow identifying the creative. Consider including a small string identifying the ad network. This is used by AMP for reporting purposes. The value MUST NOT contain user data or personal identifiable information.
+- `window.context.getHTML (selector, attrs, callback)` is a method for obtaining a content of node from the parent window. The first parameter is css selector from which the content will be received. The second parameter - the list of tag attributes, which will be left in a result string (for example: class, id). The third parameter - the callback that receives the element content as a string.
```

Comment:
... is a method that retrieves specified node's content from the parent window which cannot be accessed directly because of security restrictions caused by AMP rules and iframe's usage. 

- `selector` is a CSS selector of the node to take content from. 
- `attrs` takes an array of tag attributes to be left in the stringified HTML representation (for instance, `['id', 'class']`). All not specified attributes will be cut off from the result string. 
- `callback` takes a function to be called when the content is ready. getHTML invokes callback with the only argument of type `string`.

Ты вроде говорил, что колбэк не нужен и функция может вернуть значение синхронно. Что-то поменялось?

Ещё я бы указал типы и привёл примеры значений, но если в доке для других методов этого нет, то наверное даже мой список лишний и можно всё записать в строчку.